### ISSUE TITLE
Control Panel Nav Bar Disappears at Low Resolution

### ISSUE THREAD

**Issue by Kethsar** [CS297Sp16/LCC_Co-op_Site#97]
Currently the nav bar on the side of the control panel will disappear if the browser resolution gets too low. This has been a known issue, but it really should be fixed.

**Comment by Chef-Code** [comment]
How about applying DRY to the `<ul>` tags... and only using 2.  there is no need to have a pair for each set. This should make it easier to force the appropriate behavior out of the list... no matter the size of the screen.

**Comment by Chef-Code** [comment]
@{
    Layout = "~/Views/Shared/_Layout.cshtml"; 
}
<div class="row">
    <div class="col-sm-3 col-md-2 sidebar">
        <img src="~/Images/logo.png" class="img-responsive " />
        <ul>
        @if (User.IsInRole("Student"))
        {
            <li><h4>Student</h4></li>
                <li>@Html.ActionLink("Update your Info", "UpdateStudent")</li>

        }
        @if (User.IsInRole("Coordinator") || User.IsInRole("Admin"))
        {
            <li><h4>Coordinator</h4></li>
                <li>@Html.ActionLink("Invite Student", "InviteStudent", "ControlPanel")</li>
                <li>@Html.ActionLink("Active Student Invitations", "Invitations", "Student")</li>
                <li>@Html.ActionLink("View Majors", "", "Major")</li>
                <li>@Html.ActionLink("View Departments", "", "Department")</li>
                <li>@Html.ActionLink("View Students", "", "Student")</li>
				<li>@Html.ActionLink("Applications", "Applications", "Coop")</li>

        }
        @if (User.IsInRole("Admin"))
        {
            <li><h4>Administrator</h4></li>
                <li>@Html.ActionLink("Invite Coordinator", "InviteCoordinator", "ControlPanel")</li>
                <li>@Html.ActionLink("Active Coordinator Invitations", "Invitations", "Coordinator")</li>
                <li>@Html.ActionLink("View Coordinators", "", "Coordinator")</li>
                <li>@Html.ActionLink("Disable Coordinators", "DisableCoordinators", "ControlPanel")</li>
                <li>@Html.ActionLink("Enable Coordinators", "EnableCoordinators", "ControlPanel")</li>
                <li>@Html.ActionLink("Set SMTP Information", "SMTP", "ControlPanel")</li>
        }
            <li>@Html.ActionLink("Change Your Password", "ChangePassword", "Auth")</li>
        </ul>
    </div>
</div>

    <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main"  id="main-wrapper">
            @RenderBody()
    </div>

**Comment by Chef-Code** [comment]
```
@{
    Layout = "~/Views/Shared/_Layout.cshtml"; 
}
<div class="row">
    <div class="col-sm-3 col-md-2 sidebar">
        <img src="~/Images/logo.png" class="img-responsive " />
        <ul>    @*I just added them above and below*@
        @if (User.IsInRole("Student"))
        {
            <li><h4>Student</h4></li>  @*And placed list tags around the header tags*@
                <li>@Html.ActionLink("Update your Info", "UpdateStudent")</li>

        }
        @if (User.IsInRole("Coordinator") || User.IsInRole("Admin"))
        {
            <li><h4>Coordinator</h4></li>
                <li>@Html.ActionLink("Invite Student", "InviteStudent", "ControlPanel")</li>
                <li>@Html.ActionLink("Active Student Invitations", "Invitations", "Student")</li>
                <li>@Html.ActionLink("View Majors", "", "Major")</li>
                <li>@Html.ActionLink("View Departments", "", "Department")</li>
                <li>@Html.ActionLink("View Students", "", "Student")</li>
        <li>@Html.ActionLink("Applications", "Applications", "Coop")</li>

        }
        @if (User.IsInRole("Admin"))
        {
            <li><h4>Administrator</h4></li>
                <li>@Html.ActionLink("Invite Coordinator", "InviteCoordinator", "ControlPanel")</li>
                <li>@Html.ActionLink("Active Coordinator Invitations", "Invitations", "Coordinator")</li>
                <li>@Html.ActionLink("View Coordinators", "", "Coordinator")</li>
                <li>@Html.ActionLink("Disable Coordinators", "DisableCoordinators", "ControlPanel")</li>
                <li>@Html.ActionLink("Enable Coordinators", "EnableCoordinators", "ControlPanel")</li>
                <li>@Html.ActionLink("Set SMTP Information", "SMTP", "ControlPanel")</li>
         }
            <li>@Html.ActionLink("Change Your Password", "ChangePassword", "Auth")</li>
        </ul>
     </div>
</div>

    <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main"  id="main-wrapper">
            @RenderBody()
    </div>
```

**Comment by Kethsar** [comment]
@Chef-Code supposedly @BenMiddleton  implemented something already. That's why I assigned it to him. Check pull request #100 and talk to him about it, if you can find him.

**Comment by Chef-Code** [comment]
I know I just wanted to look smart.

**Comment by Kethsar** [comment]
Fixed with pull request #100
